mmtrack/evaluation/metrics/tao_metrics.py

```diff
@@ -137,12 +149,24 @@ def compute_metrics(self, results: List) -> Dict[str, float]:
         """
         logger: MMLogger = MMLogger.get_current_instance()
         # split gt and prediction list
-        tmp_pred_results, tmp_gt_results = zip(*results)
-        gt_results = self.format_gts(tmp_gt_results)
-        pred_results = self.format_preds(tmp_pred_results)
+        tmp_pred_track_results, tmp_pred_det_results, \
+            tmp_gt_results, tmp_meta_info = zip(*results)
+        tao_meta_info = self.format_meta(tmp_meta_info)
+        gt_results = self.format_gts(tmp_gt_results, tao_meta_info)
+        pred_track_results = self.format_preds(tmp_pred_track_results)
+        pred_det_results = self.format_preds(tmp_pred_det_results)
+
+        # save the json result to tmp dir
+        tmp_dir = tempfile.TemporaryDirectory()
+        pred_det_results_path = f'{tmp_dir.name}/tao_bbox.json'
+        gt_results_path = f'{tmp_dir.name}/tao_gt.json'
+        # LVIS api only supports reading from files
+        mmcv.dump(pred_det_results, pred_det_results_path)
+        mmcv.dump(gt_results, gt_results_path)
```

Comment:
put these codes under `if 'bbox' in self.metrics:`, since it is only used in LVIS eval